ex13.js

```diff
@@ -0,0 +1,9 @@
+function invert(array) {
+    let invertedArray = []
+    for (let i=0; i<array.length; ++i){
+        const number = array[i]
+        const invertedNumber = -number
```

Comment:
although it works it is better to use `-1 * ...` as this is easier to spot in case of a bug :)